light/rpc/client.go

```diff
@@ -462,6 +463,18 @@ func (c *Client) DataCommitment(
 	return c.next.DataCommitment(ctx, beginBlock, endBlock)
 }
 
+// DataRootInclusionProof calls rpcclient#DataRootInclusionProof method and returns
+// a merkle proof for the data root of block height `height` to the set of blocks
+// defined by `beginBlock` and `endBlock`.
```

Comment:
[question] is this a Merkle proof that proves the range of blocks [beginBlock:endBlock] exist in a Merkle tree where the root is data root? What Merkle tree is this? Does it exist in vanilla Tendermint?

Reply:
This is related to the QGB. 
The QGB contract receives commitments to a set of data roots, i.e. data commitments.  
This proof proves that a certain data root was part of the commitment (that was posted to QGB), which is defined by a range of blocks.

Reply:
I am also a bit confused as to what the arguments `beginBlock` and `endBlock` mean. These are already loaded terms: https://github.com/celestiaorg/celestia-core/blob/a952ab34f0c2f691941c0c05729dcb34411ada7e/abci/types/application.go#L24-L26

Reply:
Could we instead use something like `firstBlock` and `lastBlock`?

Reply:
[01cb80b](https://github.com/celestiaorg/celestia-core/pull/878/commits/01cb80bea07aef0f5da1e7457226e44b9fcd89fc)